File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/Step40_Parallel_LambdaA.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step40_Parallel_LambdaA extends SfnLambda<ParentPayload> {
    @Override
    public ParentPayload execute(ParentPayload payload) {
        payload.setParallelA("Foo");
        return payload;
    }
}

File: myapp/parent/Step40_Parallel_LambdaA.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.parent;

public class Step40_Parallel_LambdaA : SfnLambda<ParentPayload> {
  public override ParentPayload Execute(ParentPayload payload) {
    payload.ParallelA = "Foo";
    return payload;
  }
}
